-------------Warn at 01/09/2019 06:37:50------------- 
 Stump.Server.BaseServer.ServerBase.Initialize -> 
	Exceptions not handled cause Debugger is attatched 
-------------Warn at 01/09/2019 06:37:50------------- 

-------------Error at 01/09/2019 06:38:00------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 01/09/2019 06:38:00------------- 

-------------Warn at 01/09/2019 06:39:46------------- 
 Stump.Server.BaseServer.ServerBase.Initialize -> 
	Exceptions not handled cause Debugger is attatched 
-------------Warn at 01/09/2019 06:39:46------------- 

-------------Error at 01/09/2019 06:39:54------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 01/09/2019 06:39:54------------- 

-------------Warn at 01/09/2019 06:44:24------------- 
 Stump.Server.BaseServer.ServerBase.Initialize -> 
	Exceptions not handled cause Debugger is attatched 
-------------Warn at 01/09/2019 06:44:24------------- 

-------------Error at 01/09/2019 06:44:32------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 01/09/2019 06:44:32------------- 

-------------Warn at 01/09/2019 06:45:05------------- 
 Stump.Server.BaseServer.ServerBase.Initialize -> 
	Exceptions not handled cause Debugger is attatched 
-------------Warn at 01/09/2019 06:45:05------------- 

-------------Error at 01/09/2019 06:45:13------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 01/09/2019 06:45:13------------- 

-------------Warn at 01/09/2019 15:00:22------------- 
 Stump.Server.BaseServer.ServerBase.Initialize -> 
	Exceptions not handled cause Debugger is attatched 
-------------Warn at 01/09/2019 15:00:22------------- 

-------------Error at 01/09/2019 15:00:31------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 01/09/2019 15:00:31------------- 

-------------Warn at 01/09/2019 15:01:13------------- 
 Stump.Server.BaseServer.ServerBase.Initialize -> 
	Exceptions not handled cause Debugger is attatched 
-------------Warn at 01/09/2019 15:01:13------------- 

-------------Error at 01/09/2019 15:01:20------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 01/09/2019 15:01:20------------- 

-------------Warn at 01/09/2019 16:11:00------------- 
 Stump.Server.BaseServer.ServerBase.Initialize -> 
	Exceptions not handled cause Debugger is attatched 
-------------Warn at 01/09/2019 16:11:00------------- 

-------------Error at 01/09/2019 16:11:08------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 01/09/2019 16:11:08------------- 

-------------Error at 01/09/2019 16:19:14------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): There is already an open DataReader associated with this Connection which must be closed first.
   en MySql.Data.MySqlClient.ExceptionInterceptor.Throw(Exception exception)
   en MySql.Data.MySqlClient.MySqlConnection.Throw(Exception ex)
   en MySql.Data.MySqlClient.MySqlCommand.CheckState()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 95
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/09/2019 16:19:14------------- 

-------------Warn at 01/09/2019 23:35:48------------- 
 Stump.Server.BaseServer.ServerBase.Initialize -> 
	Exceptions not handled cause Debugger is attatched 
-------------Warn at 01/09/2019 23:35:48------------- 

-------------Error at 01/09/2019 23:35:58------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 01/09/2019 23:35:58------------- 

-------------Warn at 01/09/2019 23:56:12------------- 
 Stump.Server.BaseServer.ServerBase.Initialize -> 
	Exceptions not handled cause Debugger is attatched 
-------------Warn at 01/09/2019 23:56:12------------- 

-------------Error at 01/09/2019 23:56:21------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 01/09/2019 23:56:21------------- 

